conda-recipes/D47188-svml.patch

```diff
@@ -154,23 +122,28 @@ index 0000000..f68c086
 +def exp10      : SvmlVariant;
 +def exp2       : SvmlVariant;
 +def expm1      : SvmlVariant;
-+def floor      : SvmlVariant;
-+def fmod       : SvmlVariant;
 +def hypot      : SvmlVariant;
 +def invsqrt    : SvmlVariant;
 +def log10      : SvmlVariant;
 +def log1p      : SvmlVariant;
 +def log2       : SvmlVariant;
-+def logb       : SvmlVariant;
-+def nearbyint  : SvmlVariant;
-+def rint       : SvmlVariant;
-+def round      : SvmlVariant;
 +def sind       : SvmlVariant;
 +def sinh       : SvmlVariant;
 +def sqrt       : SvmlVariant;
 +def tan        : SvmlVariant;
 +def tanh       : SvmlVariant;
-+def trunc      : SvmlVariant;
++
++// While SVML doesn't provide _ha versions of
++// the following symbols let's disable their vectorization.
```

Comment:
Is this conveying the following, "SVML does not provide _ha version of the following symbols so prevent them from being vectorized"?

Reply:
yes, in order to speedup publishing the patch upstream, we decided to do it in smaller steps.